### PR TITLE
Removing WCF from the automatic PRs for version updates.

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
* Since WCF cannot move to the latest buildtools until other issues are resolved the PRs being generated cannot be merged and will always fail CI tests.

### CODE REVIEW

**Comment by StephenBonikowsky** [comment]
@dagood can you review this?

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -47,12 +47,6 @@
       "vsoInstance": "devdiv.visualstudio.com",
       "vsoProject": "DevDiv",
       "buildDefinitionId": 5543
-    },
-    // A build definition capable of running any .ps1 script in the WCF repo. By default, the master branch.
-    "wcf-general": {
```
Comment (dagood) [inline Maestro/subscriptions.json]: I'd prefer to leave this in, just to make sure we keep track of it.

Comment (StephenBonikowsky) [inline Maestro/subscriptions.json]: Ok

**Comment by StephenBonikowsky** [comment]
Is this good now?

**Comment by dagood** [comment]
Yep, thanks. Merging.

**Pull request merged**